### PR TITLE
solved bug on cuppertino bottom navigation bar

### PR DESCRIPTION
**Pull request by putrautama007**

### CODE REVIEW

**Comment by rrifafauzikomara** [comment]
Hi @putrautama007,

Thanks for fixing the bug, I really appreciate. And I have a request for the PR.

File: features/bookmark/lib/presentation/ui/bookmark_page.dart [hunk 1]
```diff
@@ -18,6 +18,8 @@ class BookmarkPage extends StatelessWidget {
   Widget _buildIos(BuildContext context) {
     return CupertinoPageScaffold(
       navigationBar: CupertinoNavigationBar(
+        heroTag: "boormark_page",
```
Comment (rrifafauzikomara) [inline features/bookmark/lib/presentation/ui/bookmark_page.dart]: Delete `heroTag`.

File: features/list_news/lib/presentation/ui/list_news_page.dart [hunk 2]
```diff
@@ -52,6 +52,8 @@ class ListNewsPage extends StatelessWidget {
   Widget _buildIos(BuildContext context) {
     return CupertinoPageScaffold(
       navigationBar: CupertinoNavigationBar(
+        heroTag: "list_new_page",
```
Comment (rrifafauzikomara) [inline features/list_news/lib/presentation/ui/list_news_page.dart]: You can delete `heroTag`.

**Review by rrifafauzikomara — CHANGES_REQUESTED** [review]

File: features/list_news/lib/presentation/ui/list_news_page.dart (hunk 2 above)
Comment (rrifafauzikomara) [reply features/list_news/lib/presentation/ui/list_news_page.dart]: @putrautama007 Can you also add `transitionBetweenRoutes: false,` into other screens that implemented `CupertinoNavigationBar`?

**Review by rrifafauzikomara — CHANGES_REQUESTED** [review]

File: features/bookmark/lib/presentation/ui/bookmark_page.dart (hunk 1 above)
Comment (putrautama007) [reply features/bookmark/lib/presentation/ui/bookmark_page.dart]: okay

File: features/list_news/lib/presentation/ui/list_news_page.dart (hunk 2 above)
Comment (putrautama007) [reply features/list_news/lib/presentation/ui/list_news_page.dart]: of course

**Review by putrautama007 — COMMENTED** [review]
remove hero tags

**Pull request merged**